java/common/src/main/scala/org/typelevel/otel4s/java/Conversions.scala

```diff
@@ -71,4 +74,29 @@ private[java] object Conversions {
     builder.build()
   }
 
+  def asyncFromCompletableResultCode[F[_]](
```

Comment:
scalafmt was not kind to this one.

Reply:
Also note that Natchez's Util doesn't wrap the argument in an effect.  I see this as more similar to Cats-Effect's future converter.  But since it would typically be called in a resource release, it's usually already "delayed" in practice.